### PR TITLE
Only small updates

### PR DESCRIPTION
**Pull request by COMTOP1**
Adding connecting to mailer message, typos, removing unnecessary things and comments

### CODE REVIEW

**Comment by COMTOP1** [comment]
Plus this will be a branch for all the changes to go into, just incase of it breaking, master isn't affected, merged to master at the end

File: .env.example [hunk 1]
```diff
@@ -7,6 +7,7 @@ WAUTH_DB_PORT=
 WAUTH_DB_USER=
 WAUTH_DB_PASS=
 WAUTH_DB_NAME=
+# Use either enabled or disabled
```
Comment (markspolakovs) [inline .env.example]: There's more possible values and `enabled` isn't a valid one (`require` is probably what you want, except that doesn't actually verify the certificate...) - might be better to link out to https://pkg.go.dev/github.com/lib/pq#section-documentation

File: permission/permissions.go [hunk 2]
```diff
@@ -1,53 +0,0 @@
-package permission
```
Comment (markspolakovs) [inline permission/permissions.go:1]: Is this just a leftover?

**Review by markspolakovs — CHANGES_REQUESTED** [review]

File: permission/permissions.go (hunk 2 above)
Comment (COMTOP1) [reply permission/permissions.go:1]: Yes that is an old one, some duplicates will come of this as the different versions happen, I should merge from pull 10 in a but

File: .env.example (hunk 1 above)
Comment (COMTOP1) [reply .env.example]: I think I tried with required but it fails, I'll try testing again

Comment (markspolakovs) [reply .env.example]: > Valid values for sslmode are:
> 
> disable - No SSL
> require - Always SSL (skip verification)
> verify-ca - Always SSL (verify that the certificate presented by the server was signed by a trusted CA)
> verify-full - Always SSL (verify that the certification presented by the server was signed by a trusted CA and the server host name matches the one in the certificate)

Comment (COMTOP1) [reply .env.example]: I've just updated it

**Review by markspolakovs — APPROVED** [review]

**Pull request merged**